### PR TITLE
Add qiskit-bot notification config

### PR DESCRIPTION
**Pull request by mtreinish**
### Summary

This commit adds a local config file for qiskit-bot. Qiskit-bot
recently added a feature that enables a configurable first
comment/notification (see https://github.com/Qiskit/qiskit-bot/pull/18).
This feature works by adding a local in-repo configuration file that
lists regex keys that match the paths modified in a PR when it is
opened. If the paths match any of the regexes the listed users are
mentioned in the comment the bot leaves. For this commit I picked an
initial selection of users to notify we can easily modify this and add
any user (whether they're a core team member, codeowner, or just an
individual contributor), the bot will simply parrot whatever we put in
this config on comments when the PR matches the regex.

### Details and comments

### CODE REVIEW

File: qiskit_bot.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "utils/run_circuits":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+    "utils/quantum_instance":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+    "opflow":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+        - "@Cryosis"
```
Comment (jakelishman) [inline qiskit_bot.yaml:23]: here, and elsewhere - Julien's username is Cryoris (-ris not -sis)
```suggestion
        - "@Cryoris"
```

File: qiskit_bot.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "circuit/library":
+        - "@Cryosis"
```
Comment (jakelishman) [inline qiskit_bot.yaml:25]:
```suggestion
        - "@Cryoris"
```

**Review by jakelishman — DISMISSED** [review]
Looks fine to me.  I'll leave it without "automerge" for a little while longer to see if anybody named in these notifications would prefer not to be.

@nkanazawa1989, @levbishop, @ikkoham, @manoelmarques, @woodsp-ibm, @Cryoris, @ajavadia, @t-imamichi, @kevinhartman.

**Review by levbishop — APPROVED** [review]

File: qiskit_bot.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "utils/run_circuits":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+    "utils/quantum_instance":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+    "opflow":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+        - "@Cryoris"
```
Comment (Cryoris) [inline qiskit_bot.yaml:23]:
```suggestion
        - "@Cryoris"
    "algorithms""
        - "@manoelmarques"
        - "@woodsp-ibm"
```

**Review by Cryoris — APPROVED** [review]
LGTM!

**Pull request merged**